### ISSUE TITLE
how to used with document.querySelector

### ISSUE THREAD

**Issue by productdevbook** [vueuse/motion#56]
![image](https://user-images.githubusercontent.com/38668796/159410214-58cb7e05-5a72-4401-9931-07fcbb8f6f9b.png)

**Comment by zynth17** [comment]
this doesnt work because the component is not yet mounted.
think you should create a reactive variable with ref to the elment and use the reference to that element

**Comment by productdevbook** [comment]
@zynth17 Yes, but I need to reach the place in another place. The design is not on the same page.


I guess it needs to be integrated. ? 
https://popmotion.io/stylefire/api/styler/

**Comment by Tahul** [comment]
You should actually be able to do that, but only from a `onMounted` hook.

I'm afraid `#app` ain't available yet in setup().

If it isn't the case, then could you send a reproduction in a sandboxed environment like StackBlitz or CodeSandbox?

Thanks 🙏

**Comment by productdevbook** [comment]
Why did you turn this off, this is not working :/
